Demo/src/main/java/com/paypal/android/di/NetworkModule.kt

```diff
@@ -0,0 +1,39 @@
+package com.paypal.android.di
+
+import com.google.gson.GsonBuilder
+import okhttp3.OkHttpClient
+import org.koin.dsl.module
+import java.util.concurrent.TimeUnit
+
+private const val AUTH_HEADER_VALUE_START = "Token "
+
+private const val CONNECT_TIMEOUT_IN_SEC = 20L
+private const val READ_TIMEOUT_IN_SEC = 30L
+private const val WRITE_TIMEOUT_IN_SEC = 30L
```

Comment:
Just curious - how did we decide on these timeout values? Are they standard?

Reply:
These are just values I have previously used, is there any PayPal standard times so I can update them?

Reply:
Not sure of standard times - in Braintree currently we use 30 seconds for all, but 20 is good with me to

Reply:
We have 60-second timeouts in the PayPal Checkout SDK, but I don't think there's a company-wide standard. I think what you have in the demo module is good.